.github/workflows/tests_and_qc_frontend.yml

```diff
@@ -20,14 +18,14 @@ jobs:
     - uses: actions/setup-node@v3
       with:
         node-version: 18
-    # TODO: figure out: is it possible just to cd frontend once?
-#    - name: cd
-#      run: cd frontend
     - name: Install dependencies
-#      run: cd frontend && npm ci
+#      run: cd frontend && npm ci  # only playwright is necessary to install
       run: cd frontend && npm install playwright
     - name: Install Playwright browsers
       run: cd frontend && npx playwright install --with-deps
+    - name: Re-configure for non-local web server
+      run: |
+        cd frontend && sed -n '/webServer: \[/,/\]/!p' playwright.config.js > tmp && mv tmp playwright.config.js
```

Comment:
The bugfix: I forgot I needed to remove this for the GH action. It tries to set up local sever. We eventually do want to test local inside the action, but there are other barriers to that. So for now we just want to test dev/prod, so we need to remove this local webServer config before running playwright.